docs/server-options.md

```diff
@@ -0,0 +1,111 @@
+---
+id: server-options
+title: Temporal server options
+sidebar_label: Start options
+---
+
+## Overview
+
+To run the Temporal, include the temporal server package `go.temporal.io/server/temporal` in your Go application. Then use it to create and start a new server:
```

Comment:
we need to start positioning Temporal to be usable by people who don't use Go eventually. suggest this wording

```suggestion
To run Temporal as a production Go application, include the temporal server package `go.temporal.io/server/temporal`. Then use it to create and start a new server:
```